src/pages/index.html

```diff
@@ -0,0 +1,107 @@
+          fill="#3250BD" />
+      </svg>
+    </h2>
+    <ul>
+      <li><a class="nav_link">Home</a></li>
+      <li><a class="nav_link">Moon</a></li>
+    </ul>
+  </nav>
+  <main class="main">
+    <section class="primary_section">
+      <div class="squares">
+        <div class="square_primary"></div>
+        <div class="smaller_squares">
+          <div class="square_1"></div>
+          <div class="square_2"></div>
+        </div>
+        <div class="buttons">
+          <button class="cover_button active_cover_button" type="button"></button>
+          <button class="cover_button" type="button"></button>
+          <button class="cover_button" type="button"></button>
+        </div>
+      </div>
+      <div class="computer_row">
+        <h2 class="primary_title title">The purpose of the application</h2>
+        <p class="primary_paragraph">
+          Lorem Ipsum dolor sit amet, consectetur adipiscing elit, sed do
+          eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad
+          minim veniam, quis nostrud exercitation ullamco laboris nisi ut
+          aliquip ex ea commodo consequat. Duis aute irure dolor in
+          reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla
+          pariatur. Excepteur sint occaecat cupidatat non proident, sunt in
+          culpa qui officia deserunt mollit anim id est laborum.
+        </p>
+      </div>
+    </section>
+
+    <section class="secondary_section">
+      <div class="square_second"></div>
```

Comment:
To be frankly honest, that's not a square, you should name it as something like `cover image` or `picture`.